packages/expo-camera/src/Camera.tsx

```diff
@@ -50,6 +50,10 @@ function ensureRecordingOptions(options?: RecordingOptions): RecordingOptions {
     recordingOptions.quality = Camera.Constants.VideoQuality[recordingOptions.quality];
   }
 
+  if (recordingOptions.videoBitrate && typeof recordingOptions.videoBitrate !== 'number') {
+    console.warn('Video Bitrate should be a positive integer');
```

Comment:
Remove that checking. `Typescript` is ensuring proper typings, we do not want to duplicate its work in runtime.